test/test_testing.py

```diff
@@ -6813,6 +6833,48 @@ def _on_pebble_custom_notice(self, event: ops.PebbleCustomNoticeEvent):
         assert id != ''
         assert num_notices == 0
 
+    def test_check_failed(self, request: pytest.FixtureRequest, monkeypatch: pytest.MonkeyPatch):
+        harness = ops.testing.Harness(
+            ContainerEventCharm,
+            meta="""
+            name: notifier
+            containers:
+              foo:
+                resource: foo-image
+        """,
+        )
+        request.addfinalizer(harness.cleanup)
+        harness.set_can_connect('foo', True)
+        harness.begin()
+        harness.charm.observe_container_events('foo')
+
+        def get_change(_: ops.pebble.Client, change_id: str):
+            return ops.pebble.Change.from_dict({
+                'id': change_id,
+                'kind': pebble.ChangeKind.PERFORM_CHECK.value,
+                'summary': '',
+                'status': pebble.ChangeStatus.ERROR.value,
+                'ready': False,
+                'spawn-time': '2021-02-10T04:36:22.118970777Z',
```

Comment:
I'm curious about the significance of this timestamp 😆

Reply:
I was too lazy to generate one myself so copied it from one of the existing tests. I would guess it's from when they were added.